pikaur/install_info_fetcher.py

```diff
@@ -349,7 +349,7 @@ def get_repo_pkgs_info(self) -> None:
                 self.repo_packages_install_info.append(pkg_update)
             else:
                 self.thirdparty_repo_packages_install_info.append(pkg_update)
-
+    
```

Comment:
this

Reply:
yes, I always have to wait for the CI to process it :D

Reply:
you can run `./maintenance_scripts/lint.sh` locally